src/core/data.jl

```diff
@@ -199,6 +199,127 @@ function build_mc_fault_study(data::Dict{String,<:Any}; resistance::Real=0.01, p
 end
 
 
+"""
+    build_mc_sparse_fault_study(data::Dict{String,<:Any}; resistance::Real=0.01, phase_resistance::Real=0.01)::Dict{String,Any}
+
+Builds sparse collection of fault studies using a network graph
+"""
+function build_mc_sparse_fault_study(data::Dict{String,<:Any}; resistance::Real=0.01, phase_resistance::Real=0.01)::Dict{String,Any}
+    fault_studies = Dict{String,Any}()
+    fault_bus_ids = Set()
+    vsource_bus_ids = Set([vsource["bus"] for (_,vsource) in get(data, "voltage_source", Dict())])
+
+    bus_ids = Dict(enumerate(keys(data["bus"])))
+    bus_nums = Dict([k => i for (i,k) in enumerate(keys(data["bus"]))])
+    g = LightGraphs.SimpleGraph(length(bus_nums))
```

Comment:
Maybe I'm missing it, but where does the graph `g` get used later on? I see the building of the graph, but not any explicit usage of the graph once built

Reply:
I found it